Tests/Client/Unit Tests/AtomicTests.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  AtomicTests.swift
+//  StreamChatClientTests
+//
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+class AtomicTests: XCTestCase {
+
+    func test_Atomic_whenValueChanged_callsDidSetCallback() {
+        // Setup
+        let initialValue = "Luke"
+        let newValue = "Leia"
```

Comment:
This PR is not opened on May 4th, hence this is invalid now 😂

Reply:
I find StarWars to be an excellent source of "dummy" data! So many names of characters, planets, ships, etc ...

Reply:
Love it! To turn up the creativity to 11:
```swift
let initialValue = "Anakin"
let newValue = "Darth Vader"
```
😂

Reply:
I think we need a new "spoiler alert" badge for our PR now!